tfhe/src/core_crypto/backward_compatibility/entities/gsw_ciphertext.rs

```diff
@@ -0,0 +1,8 @@
+use tfhe_versionable::VersionsDispatch;
+
+use crate::core_crypto::prelude::{Container, GswCiphertext};
+
+#[derive(VersionsDispatch)]
+pub enum GswCiphertextVersions<C: Container> {
+    V0(GswCiphertext<C>),
+}
```

Comment:
we can skip this one for now, it cannot be built by users

Reply:
done